scripts/build/Dockerfile.archlinux

```diff
@@ -0,0 +1,42 @@
+FROM docker.io/library/archlinux:latest
+
+ARG CC=gcc
+
+RUN pacman -Syu --noconfirm \
+	$CC \
+	bash \
+	make \
+	coreutils \
+	git \
+	gnutls \
+	libaio \
+	libcap \
+	libnet \
+	libnl \
+	nftables \
+	pkgconfig \
+	protobuf-c \
+	protobuf \
+	python-pip \
+	python-protobuf \
+	which \
+	sudo \
+	iptables \
+	nftables \
+	iproute2 \
+	tar \
+	bash \
+	go \
+	python-yaml \
+	flake8 \
+	asciidoctor \
+	python-junit-xml
```

Comment:
> ####### Test zdtm/static/bridge FAIL at Deps check /usr/bin/diff failed ########

The [diffutils](https://archlinux.org/packages/core/x86_64/diffutils/) package should provide `/usr/bin/diff`